Resources/Prototypes/Catalog/LatheRecipes/cooking.yml

```diff
@@ -5,3 +5,43 @@
   completetime: 500
   materials:
     steel: 90
+
+- type: latheRecipe
+  id: KitchenKnife
+  icon: Objects/Weapons/Melee/kitchen_knife.rsi
+  result: KitchenKnife
+  completetime: 400
+  materials:
+    steel: 30
+
+- type: latheRecipe
+  id: DrinkBottleOfNothingFull
+  icon: Objects/Consumable/Drinks/bottleofnothing.rsi
+  result: DrinkBottleOfNothingFull
+  completetime: 200
+  materials:
+    glass: 30
```

Comment:
This is not a recipe we want in the autolathe, since it's a bottle that the mime spawns with full of a special chem. Please remove it.

Reply:
ok. I think the bottle is gone now. Thought it was an empty container.